source/sessions/tests/unified/snapshot-sessions.json

```diff
@@ -0,0 +1,706 @@
+{
```

Comment:
`snapshot-sessions.yml` will be added after json test version will be finalized.